.spelling

```diff
@@ -1,6 +1,16 @@
+Bodgenhurst
+Brokenhurst
+Collyer's
 element14
 Farnell
 kickstart
+knockouts
+Livestream
+Mecanum
+nd
+Noo
+Pevril
```

Comment:
Pevril -> Peveril

There are few instances where this incorrect spelling is used in the text; rather than attempting to find and point them all out individually, I'll just let the spell checker find them :)

Reply:
Thanks, f687bd6